src/Discord/Internal/Rest/Interactions.hs

```diff
@@ -13,7 +13,6 @@ import Discord.Internal.Types.Interactions
 import Network.HTTP.Client.MultipartFormData (PartM, partBS)
 import Network.HTTP.Req ((/:))
```

Comment:
Maybe it's easier to add `(/~)` in the import list here, and replace the uses of `R./~`? The qualified operator isn't the prettiest syntax.